src/Atmos/Model/turbulence.jl

```diff
@@ -106,12 +106,64 @@ end
 function turbulence_tensors(m::ConstantViscosityWithDivergence,
     state::Vars, diffusive::Vars, aux::Vars, t::Real)
 
+  FT = eltype(state)
   S = diffusive.turbulence.S
   ν = m.ρν / state.ρ
   τ = (-2*ν) * S + (2*ν/3)*tr(S) * I
   return ν, τ
 end
 
+"""
+    ConstantViscousSponge <: TurbulenceClosure
+
+Turbulence with constant dynamic viscosity (`ρν`).
+Divergence terms are included in the momentum flux tensor.
```

Comment:
The diffusion in sponge layers is usually just horizontal, in which case you do not want to include the divergence terms.